### ISSUE TITLE
Drafts tab returns 'setting unknown property: craft\models\EntryDraft::entryId' in Craft 3.2.5

### ISSUE THREAD

**Issue by sjcallender** [engram-design/Workflow#76]
### Description
After site upgrade from Craft 2.6.8 to 3.2.5.1, now using Workflow 1.2.1, where we have two drafts for one entry (id 7300) in the Workflow process, we're getting this error when visiting the "Drafts" tab in Workflow 1.2.1.

```
Unknown Property – yii\base\UnknownPropertyException
Setting unknown property: craft\models\EntryDraft::entryId

yii\base\UnknownPropertyException: Setting unknown property: craft\models\EntryDraft::entryId in /Users/sjcallender/Sites/client/site/vendor/yiisoft/yii2/base/Component.php:209
Stack trace:
#0 /Users/sjcallender/Sites/client/site/vendor/craftcms/cms/src/base/Element.php(893): yii\base\Component->__set('entryId', '7300')
#1 /Users/sjcallender/Sites/client/site/vendor/yiisoft/yii2/BaseYii.php(546): craft\base\Element->__set('entryId', '7300')
#2 /Users/sjcallender/Sites/client/site/vendor/yiisoft/yii2/base/BaseObject.php(107): yii\BaseYii::configure(Object(craft\models\EntryDraft), Array)
#3 /Users/sjcallender/Sites/client/site/vendor/verbb/workflow/src/services/Drafts.php(28): yii\base\BaseObject->__construct(Array)
#4 /Users/sjcallender/Sites/client/site/vendor/verbb/workflow/src/controllers/BaseController.php(19): verbb\workflow\services\Drafts->getAllDrafts()
#5 [internal function]: verbb\workflow\controllers\BaseController->actionDrafts()
#6 /Users/sjcallender/Sites/client/site/vendor/yiisoft/yii2/base/InlineAction.php(57): call_user_func_array(Array, Array)
#7 /Users/sjcallender/Sites/client/site/vendor/yiisoft/yii2/base/Controller.php(157): yii\base\InlineAction->runWithParams(Array)
#8 /Users/sjcallender/Sites/client/site/vendor/craftcms/cms/src/web/Controller.php(187): yii\base\Controller->runAction('drafts', Array)
#9 /Users/sjcallender/Sites/client/site/vendor/yiisoft/yii2/base/Module.php(528): craft\web\Controller->runAction('drafts', Array)
#10 /Users/sjcallender/Sites/client/site/vendor/craftcms/cms/src/web/Application.php(299): yii\base\Module->runAction('workflow/base/d...', Array)
#11 /Users/sjcallender/Sites/client/site/vendor/yiisoft/yii2/web/Application.php(103): craft\web\Application->runAction('workflow/base/d...', Array)
#12 /Users/sjcallender/Sites/client/site/vendor/craftcms/cms/src/web/Application.php(284): yii\web\Application->handleRequest(Object(craft\web\Request))
#13 /Users/sjcallender/Sites/client/site/vendor/yiisoft/yii2/base/Application.php(386): craft\web\Application->handleRequest(Object(craft\web\Request))
#14 /Users/sjcallender/Sites/client/site/web/index.php(21): yii\base\Application->run()
#15 {main}
```

### Steps to reproduce

1. On Craft 3.2.5.1 and Workflow 1.2.1, visit the Drafts tab of the All Submissions view.

### Additional info

- Plugin version: 1.2.1
- Craft version: 3.2.5.1

**Comment by engram-design** [comment]
Yep, this needed to be completely re-done due to Craft 3.2 changes - I forgot about them. Should be fixed in [1.2.2](https://github.com/verbb/workflow/releases/tag/1.2.2).

Please note that as of Craft 3.2.8 this section is largely deprecated.

**Comment by sjcallender** [comment]
Thanks @engram-design.
